### ISSUE TITLE
Sector (pie slice) geometry

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#4754]
Requested on the forum: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/gv0yg8vxxUY

This was mentioned in our geometry roadmap (#766), but I wanted to make an issue so we can see how many users are requesting this feature

**Comment by pjcozzi** [comment]
I wrote C for this years ago we could start with.

**Comment by rahwang** [comment]
Came up on the forum again: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/Z0i7Yc1lR7E

**Comment by dzungpng** [comment]
Another request from the community forum: https://community.cesium.com/t/the-pipe-created-with-polylinevolumegeometry-is-wrong/11383/6

**Comment by mnumanuyar** [comment]
is there any development?

**Comment by srtrotter** [comment]
@mnumanuyar yes, this capability has existed in CesiumJS for a couple of years now. The link you posted is related.

@hpinkos this issue should probably be closed.

**Comment by ggetz** [comment]
While the [ability to render a "slice" of  an ellipsoid](https://sandcastle.cesium.com/?src=Partial%20Ellipsoids.html&label=Geometries) has existed, I believe this issue is requesting the ability to render a slice of a cylinder instead, which is not currently supported.

**Comment by srtrotter** [comment]
@ggetz got it, my mistake.

**Comment by algostrat** [comment]
Has anyone been able to generate this via an alternative geometry?
